src/framework/ui/checkbox/checkbox.spec.tsx

```diff
@@ -41,4 +98,47 @@ describe('@checkbox: component checks', () => {
     expect(onChange).toBeCalledWith(false);
   });
 
+  it('* indeterminate state checks 1', () => {
+    const onChange = jest.fn();
+
+    const component: RenderAPI = renderComponent({
+      indeterminate: true,
+      checked: true,
+      onChange: onChange,
+    });
+
+    fireEvent.press(component.getByType(TouchableOpacity));
+
+    expect(onChange).toBeCalledWith(false);
+  });
+
+  it('* indeterminate state checks 2', () => {
```

Comment:
on the race with branch coverage? 😄
looks like useless

Reply:
Removed, since coverage is not important to us